test/utils.ts

```diff
@@ -150,6 +150,7 @@ test('getOwnerAndRepo', t => {
 
 	Object.keys(ownerAndRepo).forEach(url => {
```

Comment:
```suggestion
	for (const [url, object] of ownerAndRepo) {
```

Reply:
`ownerAndRepo` isn't an iterable object. But I made it one and also avoided the `ts-ignore` 
fc5d463

Reply:
> ownerAndRepo isn't an iterable object. 

Objects aren't iterable, yes, but you could have used `Object.entries()` for that. Doesn't matter now though.